src/cmap/auth/gssapi.ts

```diff
@@ -15,7 +20,8 @@ import * as dns from 'dns';
 export class GSSAPI extends AuthProvider {
   auth(authContext: AuthContext, callback: Callback): void {
     const { connection, credentials } = authContext;
-    if (credentials == null) return callback(new MongoDriverError('credentials required'));
+    if (credentials == null)
+      return callback(new MongoMissingCredentialsError('credentials required'));
```

Comment:
Could this message be more descriptive to include GSSAPI?

Reply:
> `src/operations/execute_operations.ts:130 and 191`: I think this should be `MongoInvalidArgumentError`.

I left these since they seem more like they would be `MongoTransactionEror` to me, but I do see an argument for having them be `MongoInvalidArgumentError`